app/api/adapters/db/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
Are there any other adapter files you plan to add besides for the db? Would `app/api/db/adapters/...` make more sense if not?

Reply:
Got it, I assume you'll probably restructure the file utils we have into that which sort've work like that. Alright, no issue then